index.js

```diff
@@ -52,4 +52,118 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+const shop = document.querySelector(".store--item-list");
+const cart = document.querySelector(".cart--item-list");
+const totalNumber = document.querySelector(".total-number");
+
+function renderShop(){
+  shop.innerHTML = ""
+  state.items.forEach((item) => {
+    const li = document.createElement("li");
+    const div = document.createElement("div");
+    div.setAttribute("class", "store--item-icon");
+    const img = document.createElement("img");
+    img.src = `assets/icons/${item.id}.svg`;
+    img.alt = `${item.name}`;
+    const addButton = document.createElement("button");
+    addButton.innerText = "add to cart";
+    addButton.addEventListener("click", function (e) {
+
+
+    if (!item.quantity) {
+      item.quantity = 0;
+    }
+    item.quantity += 1;
+
+
+    if (!state.cart.find(({id}) => id === item.id)) {
+    
+      state.cart.push(item);
+
+    }
+  
+    cartItem();
+
+  });
+  li.append(div, addButton);
+  div.append(img);
+  shop.append(li);
+  });
+}
+renderShop()
+
+function cartItem() {
+  let totalPrice = 0;
+  cart.innerText = "";
+  state.cart.forEach((item) => {
+    const li = document.createElement("li");
+    const img = document.createElement("img");
+    img.setAttribute("class", "cart--item-icon");
+    img.src = `assets/icons/${item.id}.svg`;
+    img.alt = `${item.name}`;
+    const p = document.createElement("p");
+    p.innerText = `${item.name}`;
+    const removeButton = document.createElement("button");
+    removeButton.setAttribute("class", "quantity-btn remove-btn center");
+    removeButton.innerText = "-";
+    totalPrice += item.quantity * 0.35;
```

Comment:
could you get the price from the state rather than hard coding it as 0..35? that way if it is updated in the state then the code won't need to be changed